.github/workflows/build.yml

```diff
@@ -5,23 +5,36 @@ on:
     types: [ labeled, opened, synchronize, reopened ]
 
 jobs:
-  build:
+  build-website:
     # only run a manual build when the PR comes from Dependabot
     # as gatsby doesn't run on automated pull requests
     if: contains(github.event.pull_request.labels.*.name, 'dependencies')
-    name: Build
-
+    name: Build Website
     runs-on: ubuntu-latest
-
     steps:
       - name: Checkout
         uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
           node-version: '16'
-
+          cache: 'npm'
```

Comment:
I guess we should be using `yarn` here 👌 
https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#caching-packages-dependencies

Reply:
ha, good catch (you catch many things, thanks for the open eyes 🤩).
I would have not seen this because of the discussion yesterday in the context of dependabot